web/src/components/core/IssuesDialog.jsx

```diff
@@ -78,39 +77,13 @@ const IssueItems = ({ issues = [] }) => {
  *
  * @param {object} props
  * @param {import ("~/client/issues").ClientsIssues} props.issues
+ * @param {string} props.sectionId - A string which indicites which issues section should be highlighted.
```

Comment:
Also spot by cspell, 

```suggestion
 * @param {string} props.sectionId - A string which indicates which issues section should be highlighted.
```